### PR TITLE
Add getting started guide

### PR DESCRIPTION
**Pull request by cdrage**
Adds a getting started guide on how to use minishift + kedge as well as
minikube + kedge.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+There are three different guides depending on your container orchestrator as well as operating system.
+
+For beginners and the most compatibility, follow the __Minikube and Kedge__ guide.
+
+## Minikube and Kedge
+
+In this guide, we'll deploy a sample kedge file `httpd.yaml` to a Kubernetes cluster.
+
+Requirements:
+  - [minikube](https://github.com/kubernetes/minikube)
```
Comment (surajssd) [inline docs/getting-started.md]: put the downlaod instructions link!

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  - [kedge](https://github.com/kedgeproject/kedge)
```
Comment (surajssd) [inline docs/getting-started.md]: put the downlaod instructions link!

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+Connecting to cluster...
+Setting up kubeconfig...
+Starting cluster components...
+Kubectl is now configured to use the cluster
+```
+
+__Download the [httpd.yaml](https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/httpd/httpd.yaml) example file, or another example from the [examples](https://github.com/kedgeproject/kedge/tree/master/examples) GitHub directory:__
+
+```sh
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/httpd/httpd.yaml
```
Comment (surajssd) [inline docs/getting-started.md]: nit: `wget` is not installed in lot of machines by default, how about `curl -O`

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+```sh
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/backend.yaml
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/frontend.yaml
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/db.yaml
+```
+
+__Deploy directly to your OpenShift cluster with `kedge apply`:__
+
+Run `kedge apply -f backend.yaml -f frontend.yaml -f db.yaml` in the same directory as your example files.
```
Comment (surajssd) [inline docs/getting-started.md]: If we have only kedge files in the directory then we can always do `kedge apply -f .`, `.` being the directory all files in that directory will be taken.

File: docs/getting-started.md (hunk 1 above)
Comment (cdrage) [reply docs/getting-started.md]: If people follow the link, they'll find the installation instructions :)

File: docs/getting-started.md (hunk 2 above)
Comment (cdrage) [reply docs/getting-started.md]: If people follow the link, they'll find the installation instructions :)

File: docs/getting-started.md (hunk 3 above)
Comment (cdrage) [reply docs/getting-started.md]: You're right. Added!

File: docs/getting-started.md (hunk 4 above)
Comment (cdrage) [reply docs/getting-started.md]: IMO, I think we should keep the same.

**Comment by cdrage** [comment]
I'm going to merge this in for-now so we have a guide for getting-started during Kubecon

**Pull request merged**